### ISSUE TITLE
Multi line strings?

### ISSUE THREAD

**Issue by eldoy** [gilbert/zaml#5]
Any plans to finish this? Looks interesting. I would use it if there was support for multi line strings. Any idea how you'd want that to look?

**Comment by gilbert** [comment]
Hey @eldoy! I imagine python's multiline string syntax would do well:

```
project {
  title Add multiline strings
  descriptionHtml """
    This feature would be
    <b>great</b> to have.
  """
}
```

Triple quotes seem to be the least conflicting with any longer form text content one would want to write (unless you're writing python code within zaml, I suppose!). Perhaps most notably, it keeps clear of markdown and html.

The other aspect to consider is whitespace. As a developer, I would not want the above to convert to this...

    "\n    This feature would be\n    <b>great</b> to have.\n"

The extra indentation whitespace would be annoying to deal with. By default, it should behave like [Ruby's squiggly heredoc](https://infinum.com/the-capsized-eight/multiline-strings-ruby-2-3-0-the-squiggly-heredoc). The [dedent](https://github.com/dmnd/dedent) package may be a solution for this.

**Comment by eldoy** [comment]
@gilbert Yeah, that makes sense, it's also what TOML has. Agree about the white space, that is what annoys me with backtick strings in Javascript, indent should be visual only.

**Comment by gilbert** [comment]
[Multiline strings are in!](https://gilbert.github.io/zaml/editor.html#s=N4IgzgxgFgpgtgQxALhMALgS3QGxgGgBMZIAnTABywHsA7AXxH3GoFdSIYUQtcYACAJpt8QtqTAwcAM1EJahfgFUAOgpIRyVTHX4qQ+w7X78AgvzDpqpAJ57jJ-gDoXakwgBGbdPxvj+cJikpNZOaoYGBrQg9EA) Let me know if you need a case I didn't cover.

**Comment by eldoy** [comment]
@gilbert Oh nice, that was quick! I'll check it out. Thanks!
